content/fr/docs/setup/pick-right-solution.md

```diff
@@ -223,6 +223,7 @@ Ci-dessous vous trouverez un tableau récapitulatif de toutes les solutions list
 | tous                                           | tous                                                                         | multi-support                                                            | tout les CNI                                                                                                     | [docs](/docs/setup/independent/create-cluster-kubeadm/)                                       | Project ([SIG-cluster-lifecycle](https://git.k8s.io/community/sig-cluster-lifecycle))                                                                                                                                                          |
 | Google Kubernetes Engine                       |                                                                              |                                                                          | GCE                                                                                                              | [docs](https://cloud.google.com/kubernetes-engine/docs/)                                      | Commercial                                                                                                                                                                                                                                     |
 | Docker Enterprise                              | personnalisé                                                                 | [multi-support](https://success.docker.com/article/compatibility-matrix) | [multi-support](https://docs.docker.com/ee/ucp/kubernetes/install-cni-plugin/)                                   | [docs](https://docs.docker.com/ee/)                                                           | Commercial                                                                                                                                                                                                                                     |
+| OVHcloud Managed Kubernetes Service                       |  Personnalisé                                                                           | Ubuntu                                                                        | Flannel/calico                                                                                                              | [docs](https://docs.ovh.com/gb/en/kubernetes/)                                      | Commercial                                                                                                                                                                                                                                     |
```

Comment:
for the OS it is CoreOS 2135.6.0 and not Ubuntu

Reply:
It will be Ubuntu by March 17th, I anaticipated that a little bit as PR tend to take time to get merged ;) ( http://travaux.ovh.net/?do=details&id=41773& )